esp_idf_panic_decoder/pc_address_decoder.py

```diff
@@ -11,6 +11,13 @@
 # regex matches an potential address
 ADDRESS_RE = re.compile(r'0x[0-9a-f]{8}', re.IGNORECASE)
 
+# regex matches address entries in addr2line output
+ADDR2LINE_ADDRESS_LOOKAHEAD_RE = re.compile(r'(?=0x[0-9A-Fa-f]{8}\r?\n)')
+# regex matches the line number in addr2line output
+ADDR2LINE_FILE_LINE_RE = re.compile(r'(?P<file>.*):(?P<line>\d+|\?)(?: \(discriminator \d+\))?$')
+
+# Decoded PC address trace
+PcAddressLocation = namedtuple('PcAddressLocation', ['fn', 'path', 'line'])
 
 class PcAddressDecoder:
```

Comment:
Could you please share with me what was the original output and is the new one? In text format. I cannot run this right now. I'd like to suggest (if possible) how to make this a little simpler. I'm afraid that it became very complex and hard to follow.